src/main/java/org/generationcp/middleware/domain/inventory/manager/TransactionsSearchDto.java

```diff
@@ -18,6 +18,7 @@
 	private List<Integer> transactionIds;
 	private String createdByUsername;
 	private String transactionType;
+	private String transactionStatus;
```

Comment:
transactionType and transactionStatus should not be declared as String (Integer or List<Integer> if we want to add more flexibility). I am more inclined to declare them as List<Integer> so the backend is prepared to handle one or more.

Reply:
Done